tools/README.md

```diff
@@ -2,43 +2,43 @@
 
 Tools to build pre-build binary easy.
 
- - Makefile
- - build.sh
- - genpkgname.sh
+- Makefile
+- build.sh
+- genpkgname.sh
 
 Tools to upload and update URL in package files.
 
- - chkfrog.sh
- - upfrog.sh
- - upload_github_release.sh
+- chkfrog.sh
+- upfrog.sh
+- upload_github_release.sh
 
 Tools to check URL or SHA stuff from packages files.
 
- - create_sha_list.sh
- - create_url_list.sh
+- create_sha_list.sh
+- create_url_list.sh
 
 Obsolete tools to create package.  Now, we use `crew build`.
 
- - create_package.sh
+- create_package.sh
 
 ### Usage of Makefile
 
 Make working directory on your Chromebook and copy files there.
 Then, performs `make` there.
 
-```
-$ mkdir /usr/local/work
-$ cd /usr/local/work
-$ cp your-path-for-original-tool-directory/{Makefile,build.sh,genpkgname.sh} .
-$ make -k
+```bash
+mkdir /usr/local/work
+cd /usr/local/work
+cp your-path-for-original-tool-directory/{Makefile,build.sh,genpkgname.sh} .
+make -k
 ```
 
 ### Usage of upfrog.sh
 
 First, compile jfrog CLI from `https://www.jfrog.com/getcli/`.  Then, set it up.
 After that, copy compiled pre-built binary to `release/$ARCH`.
 
-```
+```text
 $ ./tools/upfrog.sh pango-1.40.9
```

Comment:
bash instead of text?  We don't use upfrog.sh anymore since we can no longer host on bintray.

Reply:
bash highlights the output as if it were a bash script though, and its not.

Reply:
Looks like the first line is executing a shell script so I'm not sure you see what I see.

Reply:
I think we're seeing the same thing. The first line executes a shell script, and the remaining lines are output from that script. That output shouldn't be highlighted as if it was an executable bash script, and neither should the line executing that script.